### PR TITLE
KansasCity file

### LINKED ISSUE #16: GitHubTeacher - tracking workflow progress
# Workflow
- [x] Create a branch
- [x] Make commits on new branch
- [x] Open a pull request
- [x] Collaborate 🎉 
- [x] Make more commits
- [x] Merge into master

🍕 🐱

### PR DESCRIPTION
**Pull request by githubteacher**
This pull request closes #16 

This pull request: 
- Creates a file for KC
- Adds things to do 

@beardofedu What do you think? You came to Kansas City, were there any other places you'd visited (Boulevard Brewery maybe?) 

Does anyone else have suggestions? 💭 ❓

### CODE REVIEW

**Comment by beardofedu** [comment]
@githubteacher I would recommend going to Boulevard Brewery for sure. Great 🍻

File: githubteacher-kansascity.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Kansas City
+
+### Things to do
+- Shawnee Mission Park
+- Worlds of Fun
```
Comment (wordsoup) [inline githubteacher-kansascity.md:5]: 👪

File: githubteacher-kansascity.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Kansas City
+
+### Things to do
```
Comment (eric-bts) [inline githubteacher-kansascity.md:3]: @githubteacher  Is the BBQ really that good?

Comment (ghost) [inline githubteacher-kansascity.md:3]: Get some great foods.

**Pull request merged**